constants/common/mimeType.ts

```diff
@@ -0,0 +1,7 @@
+const MIME_TYPE = Object.freeze({
+  JPEG: 'image/jpeg' as const,
+  JSON: 'application/json' as const,
+  JPG: 'image/jpg' as const,
+});
```

Comment:
혹시 각 프로퍼티에 `as const`를 추가하신 특별한 이유가 있나요?? 객체에 `as const`를 추가해줘도 동일하게 각 프로퍼티가 리터럴 타입으로 추론되기 때문에 나중에 MIME type이 추가될 수 있는 것을 생각하면 객체 전체에 `as const`를 추가하는 쪽이 좀 더 편할 것 같기도 해서요!

![Screen_Shot 2024-12-26 16 19 22](https://github.com/user-attachments/assets/15a46ad4-c3a7-4669-a42b-e1e7cec386db)

```suggestion
const MIME_TYPE = Object.freeze({
  JPEG: 'image/jpeg',
  JSON: 'application/json',
  JPG: 'image/jpg',
} as const);
```

Reply:
아 객체에 `as const`를 써줘도 됐었네요. 리뷰감사합니다! resolve했습니다!